examples/01_plotting/plot_demo_more_plotting.py

```diff
@@ -26,65 +45,179 @@
                        cut_coords=1, colorbar=False,
                        title="display_mode='z', cut_coords=1, colorbar=False")
 
 ########################################
+# Now we visualize contrast map with two views - 'saggital' and 'axial' and
+# coordinates are given manually to select particular cuts in two views.
+
+# argument display_mode='xz' where 'x' for saggital and 'z' for axial view.
+# argument cut_coords should match with input number of views therefore two
+# integers should be given in a list to cut slices with coordinates
 plotting.plot_stat_map(localizer_cmap_filename, display_mode='xz',
-                       cut_coords=(36, 60),
-                       title="display_mode='xz', cut_coords=(36, 60)")
+                       cut_coords=[36, 60],
+                       title="display_mode='xz', cut_coords=[36, 60]")
 
 ########################################
+# Visualizing contrast map with 'coronal', 'saggital' views and coordinates are
+# given manually in two cuts with two views.
+
+# display_mode='yx' for coronal and saggital view and coordinates will be
+# assigned in the order of direction as [x, y, z]
 plotting.plot_stat_map(localizer_cmap_filename, display_mode='yx',
-                       cut_coords=(-27, 36),
-                       title="display_mode='yx', cut_coords=(-27, 36)")
+                       cut_coords=[-27, 36],
+                       title="display_mode='yx', cut_coords=[-27, 36]")
 
 ########################################
+# Visualizing contrast map with 'coronal' and 'axial' views with manual
+# positioning of coordinates with each directional view
+
 plotting.plot_stat_map(localizer_cmap_filename, display_mode='yz',
-                       cut_coords=(-27, 60),
-                       title="display_mode='yz', cut_coords=(-27, 60)")
+                       cut_coords=[-27, 60],
+                       title="display_mode='yz', cut_coords=[-27, 60]")
 
 ###############################################################################
-# demo display objects with add_* methods
+# In second part, we switch to demonstrating various features add_* from
+# nilearn where each specific feature will be helpful in projecting brain
+# imaging results for user specific interpretation.
+
+# Import image processing tool for basic processing of functional brain image
+from nilearn import image
+
+# Compute voxel-wise mean functional image across time dimension. Now we have
+# functional image in 3D assigned in mean_haxby_img
 mean_haxby_img = image.mean_img(haxby_func_filename)
 
-# Plot T1 outline on top of the mean EPI (useful for checking coregistration)
+########################################
+# Now lets see how to use `add_edges`, method useful for checking coregistration
+# by overlaying anatomical image as edges (red) on top of mean functional image
+# (background), both being of same subject.
+
+# First, we call plotting function `plot_anat` into variable display with first
```

Comment:
we call the `plot_anat` plotting function, with a background image as first argument, in this case the mean fMRI image.